libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -116,6 +123,12 @@ static void tf_free_request(TFInferRequest *request)
     }
 }
 
+/**
+ * Create a TensorFlow inference request. All properties
+ * are initially unallocated and set as NULL.
+ *
+ * @return pointer to the allocated TFInferRequest instance.
+ */
 static TFInferRequest *tf_create_inference_request(void)
 {
     TFInferRequest *infer_request = av_malloc(sizeof(TFInferRequest));
```

Comment:
we forget to check the result, just fix it in another patch.

Reply:
Sure, thanks.

Reply:
Sure, thanks.